src/test/scala/org/broadinstitute/dsde/firecloud/MockServers.scala

```diff
@@ -45,6 +45,18 @@ object MockServers {
       attributes = Some(Map.empty)
     )
   }
+  val mockWorkspaceEntities: List[WorkspaceEntity] = {
+    List.tabulate(randomPositiveInt())(
+    n =>
+      WorkspaceEntity(
+      namespace = Some(randomAlpha()),
+      name = Some(randomAlpha()),
+      createdDate = Some(randomAlpha()),
```

Comment:
make this a date, not a string - e.g. SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZZ").format(new Date()). Randomize if you feel like it - I don't think the randomization is important right now.